docs/skopeo.1.md

```diff
@@ -47,7 +47,12 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   **ostree:**_image_[**@**_/absolute/repo/path_]
   An image in local OSTree repository.  _/absolute/repo/path_ defaults to _/ostree/repo_.
 
-## OPTIONS
+## DOCKER REPOSITORY NAMES
```

Comment:
Might be more appropriate to place this in the `skopeo-docker-list-tags` man page, at least until it becomes more generally applicable.

Reply:
I really don't want to add the letters DOCKER any more then is necessary.
CONTAINER REPORITORY NAMES
works fine here.

Reply:
It doesn’t; `/var/lib/containers` is also a “container repository”, we need to explicitly tie this to the docker/distribution format.

(This is also one minor reason to move this paragraph into the subcommand-specific man page; if it is in that one page, we won’t need to have a separate section for the syntax, so we won’t need to choose a good name for the section.)

Reply:
Please move this paragraph to `docs/skopeo-list-repository-tags.1.md`, or maybe just drop it.